app/products/playbooks/components/run_list/types.tsx

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export type RunListTabsNames = 'in-progress' | 'finished';
```

Comment:
Nitpick: I'd prefer the values to be consts that can be reused, that way we can prevent mistyping when checking against a string.